### PR TITLE
Support linux system

### LINKED ISSUE #1: Linux version or alternative
Currently this tool is only available on Windows.

Could the programmer be used in Linux as explained in [here](https://linuxconfig.org/eeprom-ch341a-programmer-read-and-write-data-to-chip-on-linux)? 

Could any owner of the programmer test it?

Regards

### PR DESCRIPTION
**Pull request by ligenxxxx**

### CODE REVIEW

**Comment by wilderk** [comment]
I tested this on Ubuntu 22.04 with the hdzero whoop lite vtx. Followed steps in the README.md. All the fields in the UI are grayed out after connecting programmer to the vtx. Not sure what is wrong.

hdzero-vtx-programmer

File: ch341_wrapper.py [hunk 1]
```diff
@@ -74,6 +75,80 @@ def dev_connect(self):
...
+        self.read_crc = 0
+        self.success = 0
+
+        try:
+            self.dll = cdll.LoadLibrary(self.dll_path)
+        except:
+            a = 1  # print("Error: Can't find"+self.dll_path)
+
+    def open_device(self):
+        self.iIndex = self.dll.CH34xOpenDevice("/dev/ch34x_pis0".encode())
```
Comment (SumolX) [inline ch341_wrapper.py:106]: I would recommend using an auto-detect method here.
```
import glob

def open_device(self):
        # Return first entry
        device = glob.glob('/dev/ch34x_pis[0-9]*')[0]
        self.iIndex = self.dll.CH34xOpenDevice(device.encode())
        return self.iIndex
```

**Comment by SumolX** [comment]
> I tested this on Ubuntu 22.04 with the hdzero whoop lite vtx. Followed steps in the README.md. All the fields in the UI are grayed out after connecting programmer to the vtx. Not sure what is wrong.
> 
> hdzero-vtx-programmer

My comment up above should resolve your issue as it will search for the first device attached to the system after you have updated the permissions to the device.  There should probably be a udev-rule that sets the permissions of the device.... but that is once an installation method has been identified.

File: ch341_wrapper.py (hunk 1 above)
Comment (SumolX) [reply ch341_wrapper.py:106]: image

**Pull request closed without merging**